src/System/Wlog/Launcher.hs

```diff
@@ -63,13 +63,17 @@ setupLogging LoggerConfig{..} = do
     liftIO $ createDirectoryIfMissing True handlerPrefix
 
     when consoleOutput $
-        initTerminalLogging isShowTime isShowTid _lcTermSeverity
+        initTerminalLogging timeFormat
+                            isShowTime
+                            isShowTid
+                            _lcTermSeverity
 
     liftIO $ setPrefix _lcFilePrefix
     processLoggers mempty _lcTree
   where
     handlerPrefix = _lcFilePrefix ?: "."
     logMapper     = appEndo _lcMapper
+    timeFormat    = fromMaybe "%Y-%m-%d %H:%M:%S%Q %Z" _lcTimeFormat
```

Comment:
How does `%S%Q` work here? Will the output string have constant length?

Reply:
No, it won't. Trailing zeroes are cutted off:

```
[node:DEBUG] [2017-10-07 13:56:46.998438 MSK] skovoroda
[node:INFO] [2017-10-07 13:56:46.998655 MSK] patak
[node:NOTICE] [2017-10-07 13:56:46.99873 MSK] boroda
[node:WARNING] [2017-10-07 13:56:46.998799 MSK] haha
[node:ERROR] [2017-10-07 13:56:46.998875 MSK] BARDAQ
[node:ERROR] [2017-10-07 13:56:46.999058 MSK] BARDAQ
```

Reply:
Me and Ivan voted for centiseconds, can you please abstract the whole thing using `formatter :: UTCTime -> Text` function instead of this formatting pattern?